### PR TITLE
Login as kubeadmin 

### PR DESCRIPTION
**Pull request by flacatus**
This will allow us to remove this line of code: https://github.com/redhat-appstudio/e2e-tests/blob/main/magefiles/installation/oauth.go

### CODE REVIEW

**Comment by flacatus** [comment]
image
Tested in https://github.com/openshift/release/pull/36008

**Review by kasemAlem — APPROVED** [review]

File: ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh [hunk 1]
```diff
@@ -27,6 +33,6 @@ echo "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com" > "${GIT_CREDS_PATH}"
 
 cd "$(mktemp -d)"
 
-git clone --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
+git clone --branch remove_oauth "https://${GITHUB_TOKEN}@github.com/flacatus/e2e-tests.git" .
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh:36]: please revert

**Review by psturc — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh (hunk 1 above)
Comment (flacatus) [reply ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh:36]: oups

**Review by psturc — APPROVED** [review]

**Review by rhopp — APPROVED** [review]

**Pull request merged**